pages/careers/_open-positions.html

```diff
@@ -1,8 +1,28 @@
 <div class="row row-page-header">
   <div class="col-xs-12">
     <h2>Open Positions</h2>
-    <p>We are always ready to add the next Grunt, but as a 100% bootstrapped company, we only hire when our revenue supports it, so timing matters. If you're inspired by what you've seen and believe in a little serendipity, we would love to hear from you. Here are the position we're looking for:</p>
+    <p>We are always ready to add the next Grunt, but as a 100% bootstrapped company, we only hire when our revenue supports it, so timing matters. If you're inspired by what you've seen and believe in a little serendipity, we would love to hear from you. Here are the position we're currently looking for:</p>
     <h3>Senior Full-Stack Engineer</h3>
+    <h4>What You'll Do</h4>
+    <ul>
+      <li><strong>Work on just about every aspect of the Gruntwork stack.</strong> Write Go-based CLI tools, work on Serverless apps in Node, write Terraform modules, write Python scripts and Bash scripts, do frontend development, learn the newest tech from AWS, invent new ways of managing infrastructure. We're looking for true full-stack engineers who can go deep in any area. We're less concerned with prior experience than we are with demonstrated ability to go deep when necessary.</li>
+    </ul>
+    <h4>Your Ideal Background</h4>
+    <ul>
+      <li>Demonstrated experience building good software.</li>
```

Comment:
How about:

* You know how to write code ("Dev")
* You have experience running production software ("Ops")

Reply:
+1. Replaced.